newrelic-integration-e2e/main.go

```diff
@@ -32,23 +31,27 @@ func processCliArgs() (string, string, string, logrus.Level) {
 	if *specsPathPtr == "" {
 		logrus.Fatalf("missing required spec_path")
 	}
+	if *flagRootDirPtr == "" {
+		logrus.Fatalf("missing required root_dir")
+	}
 	logLevel := logrus.InfoLevel
 	if *verboseModePtr {
 		logLevel = logrus.DebugLevel
 	}
-	return *licenseKeyPtr, *specsPathPtr, *agentDir, logLevel
+	return *licenseKeyPtr, *specsPathPtr, *flagRootDirPtr, *agentDir, logLevel
 
 }
 
 func main() {
 	logrus.Info("running executor")
 
-	licenseKey, specPath, agentDir, logLevel := processCliArgs()
+	licenseKey, specPath, rootDir, agentDir, logLevel := processCliArgs()
 	settings, err := settings.New(
```

Comment:
settings has the very same name of the package, it can create issues

Reply:
also `spec, err := spec.ParseSpecFile(content)` in settings.go has the same but I cannot leave a comment for it